js/main.js

```diff
@@ -32,4 +42,85 @@ function submitNewEntryForm(event) {
   $placeholderImage.setAttribute('src', './images/placeholder-image-square.jpg');
 
   $formNewEntry.reset();
+
+  $mainContainer.className = 'main-container hidden';
+  $entriesP.className = 'entries-p hidden';
+
+  $entriesContainer.className = 'entries-container';
+  $entriesUl.className = 'entries-ul';
+
+  data.view = 'entries';
+
+  location.reload();
```

Comment:
You should _**never**_ use `location.reload`. It completely refreshes and rebuilds the page, which is extremely inefficient. Why are you doing this?